config/config.go

```diff
@@ -128,9 +130,16 @@ func LoadConfiguration(c *cli.Context) error {
 	viper.AddConfigPath(filepath.Join(home, rootConfigPathName, contextsPathName))
 	viper.SetConfigName(context)
 
+	if viper.GetString("cli-version") != "" {
```

Comment:
Why only output this "Welcome..." message if the version isn't set in config? I'd output it if the version is missing as well as if it's new - both indicate a new version essentially.